#####################################
#                                   #
#  @author      : 00xWolf           #
#    GitHub    : @mmsaeed509       #
#    Developer : Mahmoud Mohamed   #
#  﫥  Copyright : Mahmoud Mohamed   #
#                                   #
#####################################

--- example.txt   (original file)
+++ example.txt   (modified file)
@@ -1,5 +1,5 @@
 This is the original line 1.
 This is the original line 2.
-This is the original line 3.
+This is the new line 3.
 This is the original line 4.
 This is the original line 5.
